pkg/cfn/manager/nodegroup.go

```diff
@@ -223,7 +223,7 @@ func (c *StackCollection) mapStackToNodeGroupSummary(stack *Stack) (*NodeGroupSu
 		return nil, errors.Wrapf(err, "error getting Cloudformation template for stack %s", *stack.StackName)
 	}
 
-	cluster := getClusterName(stack)
+	cluster := getClusterNameTag(stack)
```

Comment:
Doesn't this revive #510? 

`stack` here is a nodegroup stack whose name is `<cluster name>-nodegroup-<nodegroup name>`, but then `getClusterNameTag` will end up returning a empty string.

Reply:
No, this will get just the tag, and nodegroup stack has cluster name tag.

Reply:
Seems like I'm not following your idea correctly. Anyway, running eksctl built from your branch(at 2bc7cef2f793025eca9211ffc9b1d0a19f63ff93) produces empty cluster names in `get nodegroup`:

```
$ ./eksctl get nodegroup --cluster mycluser
CLUSTER	NODEGROUP	CREATED			MIN SIZE	MAX SIZE	DESIRED CAPACITY	INSTANCE TYPE	IMAGE ID
	nodegroup1	2019-02-02T13:06:24Z	2		2		0			t3.large	ami-0f0e8066383e7a2cb
```

Reply:
Probably you wanted a dedicated `getClusterName` func for nodegroups?

Reply:
Ah, so looks like I should make a separate PR to address this issue, not as trivial as I hoped!

> Probably you wanted a dedicated `getClusterName` func for nodegroups?

Not as such, `getClusterName` was intended to be sufficient for this particular case.